src/screens/AboutScreen.tsx

```diff
@@ -1,15 +1,267 @@
+
+//Dummy About Text. Hardcoded for now, to change once data from server is available
+const introText = `Droidcon is a global conference focused on the engineering of Android applications. Droidcon provides a forum for developers to network with other developers, share techniques, announce apps and products, and to learn and teach.
+
+This three-day developer focused gathering will be held in Nairobi Kenya on August 16th to 18th 2022 and will be the largest of its kind in Africa.
+
+It will have workshops and codelabs focused on the building of Android applications and will give participants an excellent chance to learn about the local Android development ecosystem, opportunities and services as well as meet the engineers and companies who work on them.`;
+
+const profileImage: ImageSourcePropType = require("../assets/img/DummySpeakerProfilePicture.jpeg");
+
+//Temporary function to make dummy data for the organizing team
+const makeDummyData = (n = 17) => {
+	let data = [];
+	for (let i = 1; i <= n; i++) {
+		data.push({
+			id: i.toString(),
+			name: `Member ${i}`,
+			title: `Title ${i}`,
+			image: profileImage,
+		});
+	}
+	return data;
 };
 
-export default AboutScreen;
\ No newline at end of file
+const MOCK_DATA_ORGANIZING_TEAM = makeDummyData();
+
+const AboutScreen = ({
+	navigation,
+}: NativeStackScreenProps<ParamListBase, screen_names.ABOUT, undefined>) => {
+	return (
+		<SafeAreaView style={[styles.container, styles.paddingVertical]}>
+			<StatusBar
+				backgroundColor={colors.DROIDCONKE_WHITE}
+				barStyle="dark-content"
+			/>
+			<View
+				style={[
+					styles.header,
+					styles.marginVerticalSeparator,
+					styles.paddingHorizontal,
+				]}
+			>
+				<DroidconKeIcon width={150} style={styles.droidconkeIcon} />
+				<View
+					style={[
+						layoutProperties.flexRow,
+						layoutProperties.justifyBetween,
+						layoutProperties.itemsCenter,
+					]}
+				>
+					<TouchableOpacity style={styles.buttonFeedback}>
+						<Image
+							resizeMode="contain"
+							source={require("../assets/icons/SmileyIcon.png")}
+							style={styles.buttonFeedbackContentMargin}
+						/>
+						<Text
+							style={[
+								styles.buttonFeedbackText,
+								styles.buttonFeedbackContentMargin,
+							]}
+						>
+							Feedback
+						</Text>
+						<Image
+							resizeMode="contain"
+							source={require("../assets/icons/SendIcon.png")}
+						/>
+					</TouchableOpacity>
+					<TouchableOpacity style={styles.iconWrapper}>
+						<Image
+							resizeMode="contain"
+							source={require("../assets/img/profilepicture.png")}
+						/>
+					</TouchableOpacity>
+				</View>
+			</View>
+			<ScrollView showsVerticalScrollIndicator={false}>
+				<Image
+					source={require("../assets/img/about.png")}
+					resizeMode="contain"
+					style={styles.heroImage}
+				/>
+				<View style={[styles.paddingHorizontal, styles.paddingVertical]}>
+					<Text style={[styles.headingText, { marginBottom: 10 }]}>About</Text>
```

Comment:
Please define the marginBottom style separately on `StyleSheet.Create()`

Reply:
Done